app/assets/javascripts/components/features/ui/components/column_subheading.jsx

```diff
@@ -0,0 +1,13 @@
+const ColumnSubheading = ({ text }) => {
+    return (
+      <div className='column-subheading'>
+        {text}
+      </div>
+    );
+  };
+
+ColumnSubheading.propTypes = {
+  text: React.PropTypes.string.isRequired,
```

Comment:
`React.PropTypes` is deprecated. Using `PropTypes` (and `import PropTypes from 'prop-types';)`.